django-stubs/contrib/auth/__init__.pyi

```diff
@@ -23,7 +23,7 @@ def load_backend(path: str) -> ModelBackend: ...
 def get_backends() -> List[ModelBackend]: ...
 def authenticate(request: Any = ..., **credentials: Any) -> Optional[AbstractBaseUser]: ...
 def login(
-    request: HttpRequest, user: AbstractBaseUser, backend: Optional[Union[Type[ModelBackend], str]] = ...
+    request: HttpRequest, user: Optional[AbstractBaseUser], backend: Optional[Union[Type[ModelBackend], str]] = ...
```

Comment:
Can you please also specify that `backend` has a default value `= ...`?
https://github.com/django/django/blob/master/django/contrib/auth/__init__.py#L87

Reply:
Oh, it is already there! Sorry!